src/qt/dc.cpp

```diff
@@ -474,9 +474,9 @@ bool wxQtDCImpl::DoGetPixel(wxCoord x, wxCoord y, wxColour *col) const
 
     if ( col )
     {
-        wxCHECK_MSG( m_qtImage != NULL, false, "This DC doesn't support GetPixel()" );
-
-        QColor pixel = m_qtImage->pixel( x, y );
+        wxCHECK_MSG( m_qtPixmap != NULL, false, "This DC doesn't support GetPixel()" );
+        QImage image = m_qtPixmap->toImage();
```

Comment:
This kind of transformation is generally expensive. Could it be optimized like
```suggestion
        const QPixmap& pixmap1px = m_qtPixmap->copy(x, y, 1, 1);
        const QImage& image = pixmap1px.toImage();
```
..unless that `copy()` also does a transformation internally?

Reply:
I'll have a look at the QT source code and make this change if it looks like it's going to be more performant.